feature/interface/singleton/otg_tests/singleton_test/README.md

```diff
@@ -73,6 +73,37 @@ a new testbed configuration with the desired port types.
 *   Ensure inbound and outbound unicast counters are the same
 *   Ensure counters increment at the selected SAMPLE interval
 
+### RT-5.1.4 [TODO: https://github.com/openconfig/featureprofiles/issues/2338]
+#### Breakout must be explicitly configured by gNMI client
+
+*   On DUT Port-1 with a QSFP-DD 400GBASE-DR4 transceiver inserted
+*   Ensure no breakout is configured
+*   Set Port-1 port-speed to 100G
+    *   /interfaces/interface/ethernet/config/port-speed
+*   Validate that the DUT does not create breakouts implicitly and does not set the breakout speed
+    *   /components/component/port/breakout-mode/groups/group/config
+    *   /components/component/port/breakout-mode/groups/group/config/index
+    *   /components/component/port/breakout-mode/groups/group/config/breakout-speed
+*   Validate the port-speed of the interface is not changed to 100G
```

Comment:
@LimeHat @rohit-rp what do you think about this suggestion? 

```suggestion
*   Validate 
    * the configuration is accepted and `/interfaces/interface/ethernet/config/port-speed` is set to 100G.  
    * `/interfaces/interface/ethernet/state/port-speed` remains at 400G, indicating the transceiver cannot support the configured value.
    * `/interfaces/interface/state/oper-status` remains set to `UP`
 ```

The other option would be to take the interface down because of the conflict between `config` and `state` values.  But I think in this scenario, where the TRANSCEIVER only has one valid `state/port-speed` value (400G), then any conflicting value in `config/port-speed` can be ignored by the NOS and the port can stay up.

What do you think?

Reply:
I am good with the suggested change, but will wait for @LimeHat to respond

Reply:
@dplore we follow the other option you mentioned - set the interface oper-status as `DOWN`.
I.e. we always prefer an explicit intent of the operator vs a derived speed value from a transceiver, and there are certain use cases for that. 

An operator may want an interface that can be oper `UP` only with a certain speed, and if speed is mismatched - he would prefer to exclude this link from the network topology (let's take as an example another 400/100G scenario: the operator wants that all links in his network are 400G, but a DC technician connects a new IXP link with 100G transceivers. This link will be up with 1/4th of the expected capacity)

If we say that an explicit speed config can be ignored, what role does it have in the first place? This leaf will be a no-op in a majority of cases (the only exception that comes to my mind is a multi-rate transceiver, but this is rare) 

Maybe we can allow both options, and implementers can choose what to do (prefer the implicit speed or use the oper down?